src/ldf-client-ui.js

```diff
@@ -673,9 +711,102 @@ require('yasgui-yasqe/dist/yasqe.css'); // Make webpack import the css as well
       if (this._writeResult) {
         this._resultCount++;
         this._writeResult(result);
+
+        // Adding wkt points on a map
+        for (const value in result) {
+          // This code adds points to the map
+          var checkingResult = `${result[value]}`;
+          if (checkingResult.includes('http://www.openlinksw.com/schemas/virtrdf#Geometry') || checkingResult.includes('http://www.opengis.net/ont/geosparql#wktLiteral')) {
+            let wktIncludingVariable = `${result[value]}`;
+            let _anotherOne = `${value}` + 'Label';
+            // Variable that checks to toggle view the map
+            mapView = true;
+
+            let _arrayName = result[_anotherOne];
+            let _newArray = wktIncludingVariable.split('^^', 1);
+
+            let wkt_geom3 = _newArray[0];
+            let wkt_geom1 = wkt_geom3.replace(/['"]+/g, '');
+
+            let wkt1 = new Wkt.Wkt();
+            wkt1.read(wkt_geom1);
+
+            // Logic to extract  cordinates and useful data obtained results
+            var checker;
```

Comment:
What does `checker` mean? Could we use a more descriptive variable name?

Reply:
Changed to isLabelDefined